src/helper.js

```diff
@@ -11,9 +11,38 @@ export function getExpectedSortedPrices(prices, sortOrder) {
     return sortedPrices.map(price => `$${price}`);
 }
 
-// Хелпер для обчислення очікуваних імен після сортування
+// helper for calculating expected names after sorting
 export function getExpectedSortedNames(names, sortOrder) {
     return sortOrder === 'az'
         ? [...names].sort()
         : [...names].sort((a, b) => b.localeCompare(a));
 }
+
+// helper for generating three unique indexes
+export function getUniqueIndexes(productCount) {
+    const randomIndexes = new Set();
+    const uniqueIndexesCount = Math.min(3, productCount);
+    while (randomIndexes.size < uniqueIndexesCount) {
+        const randomIndex = Math.floor(Math.random() * productCount);
+        randomIndexes.add(randomIndex);
+    };
+    return randomIndexes;
+}
```

Comment:
Is this stable? How many times test fail out of let's say 20 runs? 
If it is stable, How did you manage the indexes changing situation after adding product to the cart on the inventory?

Reply:
@batmanastasiya Yes, it is stable. The indexes are stored in a “randomIndexes” object, and each index is used to select a product from the DOM. Each index is unique. Therefore, one product cannot be added to the cart twice. Therefore, we do not need to check for the “Add to cart” button.  This way, all products in the DOM are involved and the indexes do not change after adding a product to the cart.
At the moment of adding a product to the cart, the “selectedProducts” array is created. The sequence of products in the array is determined by the sequence of adding the product to the cart. And on the “Checkout StepTwo” page, the sequence of products in the DOM is also determined by the sequence of adding products to the cart.